src/Parse.cpp

```diff
@@ -2,11 +2,12 @@
 #include "Custom_Error.hpp"
 #include <filesystem>
 
-Parse::Parse(char **av) {
+Parse::Parse(char **inputArgs) {
 	this->_isSolvable = true;
+	this->_fileName = false;
 	this->_algo = ASTAR;
 	this->_heuristics = MANHATTAN;
-	ParseAV(av);
+	ParseAV(inputArgs);
```

Comment:
ParseAV is not clear, what do you think of ParseInput ?

Reply:
Outdated